docs/src/Groups/pcgroup.md

```diff
@@ -178,6 +178,16 @@ pc_group(c::GAP_Collector)
 collector(::Type{T}, G::PcGroup) where T <: IntegerUnion
 ```
 
+# Encoding and reconstruction of pc groups
+
+The following functions provide conversions between polycyclic groups and their integer encodings,
+compatible with GAP's `CodePcGroup` and `PcGroupCode`.
```

Comment:
```suggestion
The following functions provide conversions between polycyclic groups and their integer encodings.
```
This is already part of the docstring right after this sentence, so IMO this is redundant